quota/etcd/quotaapi/quota_server.go

```diff
@@ -95,17 +96,26 @@ func (s *Server) GetConfig(ctx context.Context, req *quotapb.GetConfigRequest) (
 	if err != nil {
 		return nil, err
 	}
-	return getConfig(req.Name, cfgs, codes.NotFound)
+	return s.getConfig(ctx, req.Name, cfgs, codes.NotFound)
 }
 
 // getConfig finds the Config named "name" on "cfgs", converts it to API and returns it.
 // If the config cannot be found an error with code "code" is returned.
-func getConfig(name string, cfgs *storagepb.Configs, code codes.Code) (*quotapb.Config, error) {
-	cfg, ok := findByName(name, cfgs)
+func (s *Server) getConfig(ctx context.Context, name string, cfgs *storagepb.Configs, code codes.Code) (*quotapb.Config, error) {
+	storedCfg, ok := findByName(name, cfgs)
 	if !ok {
 		return nil, status.Errorf(code, "%q not found", name)
 	}
-	return convertToAPI(cfg), nil
+	cfg := convertToAPI(storedCfg)
+
+	tokens, err := s.qs.Peek(ctx, []string{cfg.Name})
+	if err == nil {
```

Comment:
I think style is to indent the non error case and then return early. I don't like it either.

Reply:
Sorry, I meant indent the error case.

Reply:
I don't think that practice rule applies in this context; this error instance never gets forwarded, nor does it really abort the flow. We can (and do) proceed regardless of it. It's kind of a "bonus".

Reply:
I didn't see the 2nd reply in time, now I'm not sure I understood you in the first place.

Reply:
It's normally 

if err != nil {
  handle it
 return wibble
}

// non error case

Reply:
Right. I understood you correctly, then. I don't think that applies here, as this isn't the "standard" `return err` scenario: in this case err doesn't stop the flow nor ever gets returned. I could test for the negative if you prefer, although personally I dislike testing the negative when both if branches have work to do.

Reply:
Why does Peek() return an error if we don't need to handle it? Can it not just return nil tokens?

Reply:
Discussed offline.

I've bumped the log to warning and tweaked the message a bit.

TL;DR: It's OK to discard Peek() errors in this context, as it's likely that the caller can live without current_tokens.